src/main/scala/nodes/learning/KernelGenerator.scala

```diff
@@ -22,9 +22,8 @@ abstract class KernelGenerator extends Transformer[DenseVector[Double], DenseVec
   /* Cached KernelBlockBlock (used for training), and corresponding
    * blockIdx
    */
-  var cachedBlockBlock:Option[(DenseMatrix[Double], Seq[Int])] = None
 
-  def apply(testMat: RDD[DenseVector[Double]], blockIdxs: Seq[Int], same: Boolean = false, cache: Boolean = false): RDD[DenseVector[Double]]
+  def apply(testMat: RDD[DenseVector[Double]], blockIdxs: Seq[Int], same: Boolean = false): RDD[DenseVector[Double]]
```

Comment:
Ok yeah this sucks. How about for this cut. We have KernelBlockModel as a transformer from DenseVector[RDD] to DenseVector[RDD]  and have it take in the KernelGenerator as before. Trying to fix this properly has caused  much headache :(